tests/manage/pv_services/test_pvc_invalid_inputs.py

```diff
@@ -59,32 +58,54 @@ def teardown():
 
 @tier3
 @pytest.mark.usefixtures(test_fixture.__name__)
-@pytest.mark.polarion_id("OCS-284")
+@pytest.mark.parametrize(
+    argnames=['volume_mode', 'access_mode'],
+    argvalues=[
+        pytest.param(
+            *['Block', constants.ACCESS_MODE_RWX],
+            marks=[pytest.mark.polarion_id("OCS-875"), pytest.mark.polarion_id("OCS-883")]),
+        pytest.param(*['Block', constants.ACCESS_MODE_RWX],
+                     marks=pytest.mark.polarion_id("OCS-882")
+                     ),
+        pytest.param(*['None', constants.ACCESS_MODE_RWO],
+                     marks=pytest.mark.polarion_id("OCS-284")
+                     )
+    ])
 class TestPvcCreationInvalidInputs(ManageTest):
     """
     PVC creation with invaid inputs in pvc yaml
     """
-    def test_pvccreation_invalid_inputs(self):
+
+    def test_pvccreation_invalid_inputs(self, volume_mode, access_mode):
         """
         Calling functions for pvc invalid name and size
         """
-        create_pvc_invalid_name(pvcname='@123')
-        create_pvc_invalid_size(pvcsize='t@st')
+        create_pvc_invalid_name(pvcname='@123', volume_mode=volume_mode, access_mode=access_mode)
+        create_pvc_invalid_size(pvcsize='t@st', volume_mode=volume_mode, access_mode=access_mode)
+        create_pvc_invalid_namespace(pvcsize='t@st', volume_mode=volume_mode, access_mode=access_mode, namespace='xyz')
 
 
-def create_pvc_invalid_name(pvcname):
+def create_pvc_invalid_name(pvcname, volume_mode=None, access_mode=constants.ACCESS_MODE_RWO,
```

Comment:
Can we instead utilize tests/helpers.py:386 - create_pvc(), so we won't have to maintain 2 functions for PVC creation?